[2024-03-24 08:56:22,176] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: weather_data_pipeline.transform_last_20 2024-03-24T08:55:00+00:00 [queued]>
[2024-03-24 08:56:22,195] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: weather_data_pipeline.transform_last_20 2024-03-24T08:55:00+00:00 [queued]>
[2024-03-24 08:56:22,197] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-03-24 08:56:22,198] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2024-03-24 08:56:22,198] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-03-24 08:56:22,224] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): transform_last_20> on 2024-03-24T08:55:00+00:00
[2024-03-24 08:56:22,232] {standard_task_runner.py:52} INFO - Started process 18161 to run task
[2024-03-24 08:56:22,237] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'weather_data_pipeline', 'transform_last_20', '2024-03-24T08:55:00+00:00', '--job-id', '879', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/file.py', '--cfg-path', '/tmp/tmpv666znmj', '--error-file', '/tmp/tmpnjwkkhla']
[2024-03-24 08:56:22,238] {standard_task_runner.py:77} INFO - Job 879: Subtask transform_last_20
[2024-03-24 08:56:22,318] {logging_mixin.py:104} INFO - Running <TaskInstance: weather_data_pipeline.transform_last_20 2024-03-24T08:55:00+00:00 [running]> on host ce3400b6b8c4
[2024-03-24 08:56:22,417] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weather_data_pipeline
AIRFLOW_CTX_TASK_ID=transform_last_20
AIRFLOW_CTX_EXECUTION_DATE=2024-03-24T08:55:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-24T08:55:00+00:00
[2024-03-24 08:56:22,507] {python.py:151} INFO - Done. Returned value was: None
[2024-03-24 08:56:22,525] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=transform_last_20, execution_date=20240324T085500, start_date=20240324T085622, end_date=20240324T085622
[2024-03-24 08:56:22,570] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-24 08:56:22,611] {local_task_job.py:151} INFO - Task exited with return code 0
